docs/user_guide/9_changelog.md

```diff
@@ -1,6 +1,23 @@
 ### ROS Kinetic
 
-#### Version 0.2.28 (current kinetic-release) 
+#### Version 0.2.43 (current kinetic-release) 
+
+Features:
+
+* REST API new functionality:
+  * Two new calls to open rqt plugins: Joint sliders and Change controllers
+  * New calls and buttons to import, export and delete grasps from database
+  * New tab with template to create new grasp
+* Refactored the grasp database to have all the functionality only available in sr_manipulation
+* AWS client is installed inside the Docker image (for uploading ROS logs and core dumps to AWS)
+* Python and C++ core dumps have been added to ROS logs and the upload to AWS
+* New demo page has been added to REST API and the demo looping through all grasps is now available in that page
+* "Create grasp" page has been renamed to "Grasp Editor" and new grasp visualiser is available to check all information related to a grasp
+* mujoco_ros repo has been cleaned up
```

Comment:
I dont think this is part of the image right?

Reply:
I thought it was. The python and C++ core dumps are there, demo page for Rest API is there, Grasp editor is there. The only one I didn't check (because I didn't test mujoco and there was no test case for it): mujoco_ros repo cleanup.